code/particle/util/ParticleProperties.cpp

```diff
@@ -63,9 +83,14 @@ WeakParticlePtr ParticleProperties::createPersistentParticle(particle_info& info
 }
 
 void ParticleProperties::pageIn() {
-	if (m_bitmap >= 0) {
-		bm_page_in_aabitmap(m_bitmap, -1);
+	
+	size_t num_bitmaps = m_bitmap_list.size();
```

Comment:
Could also use a range based for loop here since that automatically handles the empty case properly.